src/app/auth/signup.tsx

```diff
@@ -1,8 +1,11 @@
 import { Redirect, Link, router } from 'expo-router';
-import React, { useState } from 'react';
-import { Alert, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { Alert, View, Text } from 'react-native';
 import { Button, Input } from 'react-native-elements';
 
+import Icon from '../../../assets/icons';
+import color from '../../styles/colors';
```

Comment:
Right now your PR is failing checks because you're accidentally importing from the same file twice. Delete line 7 (the wrong import) and you should be good!